FlowCrypt/Controllers/Setup/SetupImportKeyViewController.swift

```diff
@@ -171,7 +173,7 @@ extension ImportKeyViewController {
     }
```

Comment:
Let's call this class `SetupManuallyImportKeyViewController` for clarity. Also, here `parseFetched` is misleading as the key was not fetched but inputted by user. Could call it eg `parseUserProvided`